lib/wallet/wallet.js

```diff
@@ -59,6 +59,8 @@ class Wallet extends EventEmitter {
     this.writeLock = new Lock();
     this.fundLock = new Lock();
 
+    this.neutrino = false;
```

Comment:
wallet class should not care about this.

Reply:
Yeah. Will remove it.